.ado/jobs/universal.yml

```diff
@@ -83,6 +83,14 @@
                     multicoreBuild: true
                     msbuildArguments: /p:RNW_FASTBUILD=${{ matrix.FastBuild }}
 
+                - task: PowerShell@2
+                  displayName: "Binskim Analysis"
+                  condition: and(succeeded(), eq('${{ matrix.BuildConfiguration }}', 'Release'), eq('${{ matrix.FastBuild }}', 'false'), ne('${{ matrix.BuildPlatform }}', 'ARM64'))
+                  inputs:
+                      targetType: script
+                      script: .\binskim analyze $(Build.SourcesDirectory)\vnext\target\${{ matrix.BuildPlatform }}\${{ matrix.BuildConfiguration }}\Microsoft.ReactNative\Microsoft.ReactNative.dll
+                      workingDirectory: $(Build.SourcesDirectory)\vnext\packages\Microsoft.CodeAnalysis.BinSkim.1.9.0-prerelease1\tools\netcoreapp3.1\win-x64
```

Comment:
curious if we can do something smart here to not depend on the specific binskim version (and therefore need to update every time we rev binskim's package version number)

Reply:
actually, there is a github actions task we can use:
```
  - task: BinSkim@3
    inputs:
      InputType: 'Basic'
      Function: 'analyze'
      AnalyzeTarget: '$(Build.ArtifactStagingDirectory)\*.dll;$(Build.ArtifactStagingDirectory)\*.exe'
      AnalyzeVerbose: true
```

See this PR for Reunion which did similar work:  microsoft/WindowsAppSDK#600

Reply:
@JunielKatarn as FYI: If we don't go with @asklar's suggestion this will break when we drop `packages.config` and enable `<PackageRefeerence` so I'd vote for the built-in task as well.